### DISCUSSION TITLE
能不能加一个url导入图片

### DISCUSSION THREAD

**Discussion by zaizaiyou** [lsky-org/lsky-pro#388] [问答]
目前只有上传本地图片，能不能加一个外链上传。
比如一行一个
多行可以上传多个外链图片。

**Comment by 0xxb** [comment]
不能，这类似于爬虫功能了，可能会引起版权纠纷问题。不过单个 url 远程下载倒是可以考虑一下。

**Reply by Tonmyn** [reply]
把图片网址粘贴到上传栏里面，然后按钮确认上传，有这个功能就很方便了，就不用在本地再转存一次了
